### PR TITLE
Updates to RPM build

### PR DESCRIPTION
**Pull request by jw3**

### CODE REVIEW

File: .github/workflows/rpm.yml [hunk 1]
```diff
@@ -19,18 +19,18 @@ jobs:
...
-      - name: Install basic rpmbuild toolchain
-        run: |
-          dnf install -y rpm-build rpmdevtools dnf-plugins-core python3-jinja2-cli git
-
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.ref }}
           fetch-depth: 0
 
+      - name: Install basic rpmbuild toolchain
```
Comment (jw3) [inline .github/workflows/rpm.yml:27]: did this get moved intentionally?

File: .github/workflows/rpm.yml [hunk 2]
```diff
@@ -19,18 +19,18 @@ jobs:
...
+        run: |
+          dnf install -y rpm-build rpmdevtools dnf-plugins-core python3-jinja2-cli
```
Comment (jw3) [inline .github/workflows/rpm.yml:29]: git no longer installed

Comment (jw3) [reply .github/workflows/rpm.yml]: now is is

File: .github/workflows/rpm.yml (hunk 1 above)
Comment (jw3) [reply .github/workflows/rpm.yml]: moved back

**Review by tparchambault — APPROVED** [review]

**Pull request merged**